tendrl/gluster_integration/message/callback.py

```diff
@@ -597,76 +600,89 @@ def volume_remove_brick_force(self, event):
         time.sleep(self.sync_interval)
         # Event returns bricks list as space separated single string
         bricks = event['message']['bricks'].split(" ")
-        for brick in bricks:
-            fetched_brick = NS.gluster.objects.Brick(
-                fqdn=brick.split(":/")[0],
-                brick_dir=brick.split(":/")[1].replace('/', '_')
-            ).load()
-
-            try:
-                NS._int.wclient.delete(
+        try:
+            for brick in bricks:
+                # find fqdn using ip
+                ip = socket.gethostbyname(brick.split(":/")[0])
+                node_id = etcd_utils.read("indexes/ip/%s" % ip).value
```

Comment:
Ensure strictly that this `node_id` is part of the current integration_id ( i.e. part of this cluster).

Other wise this will break when managing two clusters with overlapping IP ranges